queries/haskell/rainbow-parens.scm

```diff
@@ -0,0 +1,60 @@
+(exp_apply (_
+  (("(" @opening)
+   (")" @closing)))) @container
+
+(con_unit
+  (("(" @opening)
+   (")" @closing))) @container
+
+(exports
+  (("(" @opening)
+   (")" @closing))) @container
+
+(export_names
+  (("(" @opening)
+   (")" @closing))) @container
+
+(import_item
+  (("(" @opening)
+   (")" @closing))) @container
+
+(type_parens
+  (("(" @opening)
+   (")" @closing))) @container
+
+(pat_parens
+  (("(" @opening)
+   (")" @closing))) @container
+
+(pat_tuple
+  (("(" @opening)
+   (")" @closing))) @container
+
+(type_tuple
+  (("(" @opening)
+   (")" @closing))) @container
+
+(deriving
+  (("(" @opening)
+   (")" @closing))) @container
+
+(record_fields
+  (("{" @opening)
+   ("}" @closing))) @container
+
+(exp_record
+  (("{" @opening)
+   ("}" @closing))) @container
+
+(pat_fields
+  (("{" @opening)
+   ("}" @closing))) @container
```

Comment:
There is no `pat_fields` node in the test file. Please add code with such a node.